### PR TITLE
Add Chinese translations

### PR DESCRIPTION
**Pull request by wangding**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "how-to-markdown",
-  "version": "1.2.1",
+  "version": "1.2.2",
```
Comment (denysdovhan) [inline package.json]: Do not modify version. I'll do it by myself.

File: package.json [hunk 2]
```diff
@@ -10,6 +10,9 @@
     "type": "git",
     "url": "git+https://github.com/workshopper/how-to-markdown.git"
   },
+  "contributors": [
+    "王顶 <408542507@qq.com> (https://github.com/wangding)"
```
Comment (denysdovhan) [inline package.json]: Please, use Latin characters in `package.json`

File: exercises/blockquotes/solution/zh-cn.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# 好棒！
```
Comment (denysdovhan) [inline exercises/blockquotes/solution/zh-cn.md:1]: Please, find someone who can check and verify your translation.

**Review by denysdovhan — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (wangding) [reply package.json]: ok

File: package.json (hunk 2 above)
Comment (wangding) [reply package.json]: ok

File: exercises/blockquotes/solution/zh-cn.md (hunk 3 above)
Comment (wangding) [reply exercises/blockquotes/solution/zh-cn.md:1]: ok

**Comment by wangding** [comment]
commit ac384b5
I have fixed the name and version in package.json @denysdovhan

**Comment by lilizhiwei** [comment]
I have reviewed all the commits, there is no problem. @denysdovhan

**Comment by wangding** [comment]
thank you for your patient :smile:  @lilizhiwei

**Comment by gaomengyao** [comment]
I think there is no problem in these commits. @denysdovhan

**Comment by denysdovhan** [comment]
@lilizhiwei @gaomengyao hmm, strange. Usually there are at least few mistakes in translation. Anyway, gonna merge that. PR is welcome, so if there would be mistakes anyone can fix them.

**Comment by denysdovhan** [comment]
Published as `v1.3.0`!
Thank you @wangding for contributing!
Thanks to @lilizhiwei @gaomengyao for review!

**Comment by denysdovhan** [comment]
<blockquote class="twitter-tweet" data-partner="tweetdeck"><p lang="en" dir="ltr">🇨🇳🇨🇳🇨🇳 how-to-markdown@1.3.0 is out with Chinese translation! 🇨🇳🇨🇳🇨🇳<br>Thanks to wangding (<a href="https://t.co/Df0Q3mXiSW">https://t.co/Df0Q3mXiSW</a>)! <a href="https://t.co/gO8OWUdJrr">pic.twitter.com/gO8OWUdJrr</a></p>&mdash; Denys Dovhan (@denysdovhan) <a href="https://twitter.com/denysdovhan/status/824596723859386368">January 26, 2017</a></blockquote>

**Pull request merged**